### DISCUSSION TITLE
When "Right Click Moves Camera" is enabled in the Camera Plug-in, I cannot right click certain objects...

### DISCUSSION THREAD

**Discussion by XVXboy** [runelite/runelite#16937] [Q&A]
This issue just started today. I have tried uninstalling runelite and reinstalling it to no avail. When "Right Click Moves Camera" is enabled in the Camera plug-in, a setting I have used for as long as I have used Runelite, I am no longer able to right click on objects that only have an examine option. I.e., I CAN right click on the sheep at makeover mage, but I cannot right click on the lil' lamb nor any of the plants or decorations in the mage's house. **This issue persists whether or not "Right Click Objects" is also enabled.**

I can right click objects normally when the "Right Click Moves Camera" is disabled.
I realize this is a pretty minor bug, but as someone who enjoys reading the examine text on objects, especially during quests, this is very frustrating. Any help would be appreciated.

Also apologies if this issue has been addressed elsewhere. This is my first time using Github.

**Comment by Felanbird** [comment]
enable `right click objects`
